### PR TITLE
v1.1.0 - MDC 29

### PR DESCRIPTION
**Pull request by jamesmfriedman**
Not many breaking changes from MDC this time (thankfully).

- Removed the multi-select
- documented ToolbarFixedAdjust
- documented an additional case for Enzyme testing setup
- Select now re-inits appropriately when switching between cssOnly and fully featured
- Added a global dark theme to the docs that you can toggle on and off as well as some additional docs cleanup

Anything else we should add to this release?

### CODE REVIEW

**Comment by jamesmfriedman** [comment]
@j-o-d-o @izakfilmalter I just added a more fluid fix for the testing issue people have been experiencing. Check out the latest commit. Would be great if someone could verify it works for them as well.

**Comment by jamesmfriedman** [comment]
Can anyone else get some eyes on this today? The changes are really minimal as far as breaking changes go, but there are a good amount of small fixes that have made their way in from the community.

I also want to make sure the testing polyfill will work for everyone. It at least makes life easier :).

@alecchisi @JetFault @UFOMelkor @alexeyraspopov @jrajav @j-o-d-o @izakfilmalter

**Comment by izakfilmalter** [comment]
I can look at it tomorrow. Today I'm smashed. Still need to figure out
tabbar as well.

**Comment by jamesmfriedman** [comment]
@izakfilmalter  Yeah, sorry you're still having those issues. Hit me with a repro of it if you get a chance and I'll take a look at it tonight.

**Comment by j-o-d-o** [comment]
LGTM

**Comment by JetFault** [comment]
I tested it on our site (we heavily use text fields) and all the text field changes look great. I also verified 9d886b6 was working correctly. 

Tests all completed successfully.

This also fixed a bunch of mdc textfield issues I had with mdc which is nice :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You'll need a recent version of [nodejs](https://nodejs.org/en/) to work on RMWC
 > **NOTE**: If you expect to commit updated or new dependencies, please ensure you are using npm 5, which will
 > also update `package-lock.json` correctly when you install or upgrade packages.
 
-Once node is installed, simply clone our repo (or your fork of it) and run `npm install`
+Once node is installed, simply clone our repo (or your fork of it) and run `npm install`. You'll notice you have to install react and react-dom with the no-save option. These are peer dependencies that are not installed automatically.
```
Comment (ricardobrandao) [inline CONTRIBUTING.md]: This does not apply anymore, right?

**Comment by izakfilmalter** [comment]
Testing in my app this morning. Will also try to add a fix for the `TabBar` stuff.

File: CONTRIBUTING.md (hunk 1 above)
Comment (j-o-d-o) [reply CONTRIBUTING.md]: React is still as a peer dependency in the package.json as I see it. So installing react is still needed right?

Comment (jamesmfriedman) [reply CONTRIBUTING.md]: Someone put in a PR that fixed it. Looked at other libraries, they still include React as a dev dependency which shouldn’t cause any problems when being used in a normal context with npm install.

**Comment by jamesmfriedman** [comment]
@izakfilmalter it would be awesome to simplify that clause. Basically the tab bar needs to re-init if it’s children change, but we have all the problems of checking object identity with that. 

I’m bummed you can’t repro it :/

File: CONTRIBUTING.md (hunk 1 above)
Comment (j-o-d-o) [reply CONTRIBUTING.md]: Ah, was looking at the v1.0.2 tag ... sorry.

**Comment by izakfilmalter** [comment]
Gonna pull the patch and then run my tests.

**Pull request merged**